python/grass/grassdb/checks.py

```diff
@@ -169,6 +173,30 @@ def can_start_in_mapset(mapset_path, ignore_lock=False):
     return True
 
 
+def get_reason_mapset_not_usable(mapset_path):
+    """It gets reasons when:
+     * Mapset is invalid.
+     * Mapset is owned by different user.
+     * Mapset is locked.
+
+    Returns message as string if there was a reason, otherwise None.
+    """
+    if mapset_path is None:
+        return
+    message = None
+
+    # Check whether mapset is valid
+    if not is_mapset_valid(mapset_path):
+        message = _("invalid")
+    # Check whether mapset is owned by current user
+    elif not is_current_user_mapset_owner(mapset_path):
+        message = _("owned by different user")
```

Comment:
Here, the `_()` is actually harmful since you are using it for comparison (and even not using `_()` there). All these are just internal constants or flags, so they should not be translated and they should look like ones, so this one can be something like `"different-owner"`. Sure, it can be the whole things with spaces and what not, but that be quite counter-productive. The function doc should also say the reason is indicated by these strings. It is not a message, or at least not meant to be used as one or in one unlike the other functions. Maybe the name should be different, like `reason_id`? I'm not sure.

Reply:
Makes sense. I like` reason_id`.